utils/breakpoints.js

```diff
@@ -0,0 +1,47 @@
+import { setActiveBreakpoint } from '../src/state/app.js';
+
+export const defaultBreakpoints = {
```

Comment:
in theme.js we have 5 sizes?

Reply:
Yes, we added xxlarge to support the LHS nav.

Reply:
Yep I've seen that ...but why ?